doc/reference/reference_sql/sql-features.rst

```diff
@@ -14,443 +13,1032 @@ else we will mark it "Fail".
+            -   ``SELECT count(*) FROM vv2 GROUP BY s1 HAVING count(*) > 0;``   
+            -   Okay.
+   
+   
+F181, Multiple module support   
+-----------------------------
+
+Fail. Tarantool doesn't have modules.   
+   
+   
+F201, CAST function
+-------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   F201
+            -   CAST function
+            -   ``SELECT cast(s1 AS INT) FROM t;``
+            -   :ref:`Okay <sql_function_cast>`.   
+   
+
+F221, Explicit defaults
+-----------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests    
+        *   -   F221   
+            -   Explicit defaults
+            -   ``UPDATE t SET s1 = DEFAULT;``   
+            -   Fail. Syntax error.   
+     
+
+F261, CASE expression   
+---------------------
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 10 30 30 30
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Example
+            -   Tests 
+        *   -   F261-01   
+            -   Simple CASE
+            -   ``SELECT CASE WHEN 1 = 0 THEN 5 ELSE 7 END FROM t;``   
+            -   Okay.   
+        *   -   F261-02   
+            -   Searched CASE
+            -   ``SELECT CASE 1 WHEN 0 THEN 5 ELSE 7 END FROM t;``   
+            -   Okay.   
+        *   -   F261-03   
+            -   NULLIF
+            -   ``SELECT nullif(s1,7) FROM t;``
+            -   :ref:`Okay <sql_function_nullif>`   
+        *   -   F261-04   
+            -   COALESCE
+            -   ``SELECT coalesce(s1,7) FROM t;``
+            -   :ref:`Okay <sql_function_coalesce>`.   
+  
+   
+F311, Schema definition statement
+---------------------------------
+   
+..  container:: table
+
+    ..  list-table::
+        :widths: 20 40 40
+        :header-rows: 1
+
+        *   -   Feature ID
+            -   Feature
+            -   Tests 
+        *   -   F311-01   
+            -   CREATE SCHEMA   
+            -   Fail. Tarantool doesn't have schemas or databases.   
+        *   -   F311-02   
+            -   CREATE TABLE for persistent base tables   
+            -   Fail. Tarantool doesn't have CREATE TABLE inside CREATE SCHEMA.   
+        *   -   F311-03   
+            -   CREATE VIEW   
+            -   Fail. Tarantool doesn't have CREATE VIEW inside CREATE SCHEMA.   
+        *   -   F311-04   
+            -   CREATE VIEW: WITH CHECK OPTION   
+            -   Fail. Tarantool doesn't have CREATE VIEW inside CREATE SCHEMA.   
+        *   -   F311-05   
+            -   GRANT statement   
+            -   Fail. Tarantool doesn't have GRANT inside CREATE SCHEMA.   
```

Comment:
@pgulutzan, since you're here, maybe you can help me out with this section, F311. It seems a little confusing that we first state that Tarantool doesn't have schemas, so we can't perform CREATE SCHEMA, but then we say that “Tarantool doesn't have CREATE TABLE inside CREATE SCHEMA”. If we can't perform CREATE SCHEMA, isn't it the main reason we can't comply with other items in F311 section?

Reply:
I don't see how that is confusing. Suppose someday we allow CREATE SCHEMA, but without any GRANT clauses within it. Then "Tarantool doesn't have GRANT inside CREATE SCHEMA" is still true. So CREATE SCHEMA is an essential prerequisite, but the specific thing that is missing is GRANT.

Reply:
@pgulutzan okay, got it :)